### PR TITLE
Use view binding on SuggestedEditsCardsItemFragment.kt

### PR DESCRIPTION
**Pull request by cooltey**

### CODE REVIEW

File: app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsCardsItemFragment.kt [hunk 1]
```diff
@@ -28,38 +27,41 @@ import org.wikipedia.util.log.L
...
     var sourceSummaryForEdit: PageSummaryForEdit? = null
     var targetSummaryForEdit: PageSummaryForEdit? = null
     var addedContribution: String = ""
         internal set
 
-    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
         super.onCreateView(inflater, container, savedInstanceState)
-        return inflater.inflate(R.layout.fragment_suggested_edits_cards_item, container, false)
+        _binding = FragmentSuggestedEditsCardsItemBinding.inflate(LayoutInflater.from(context), container, false)
```
Comment (dbrant) [inline app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsCardsItemFragment.kt:39]: Can we just use the `inflater` parameter that is given to us by this function?

File: app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsCardsItemFragment.kt [hunk 2]
```diff
@@ -28,38 +27,41 @@ import org.wikipedia.util.log.L
 import org.wikipedia.views.ImageZoomHelper
 
 class SuggestedEditsCardsItemFragment : SuggestedEditsItemFragment() {
+    private var _binding: FragmentSuggestedEditsCardsItemBinding? = null
```
Comment (dbrant) [inline app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsCardsItemFragment.kt:30]: Should we set this to `null` in `onDestroy()`?

**Review by dbrant — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsCardsItemFragment.kt (hunk 1 above)
Comment (cooltey) [reply app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsCardsItemFragment.kt]: Yes and done.

File: app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsCardsItemFragment.kt (hunk 2 above)
Comment (cooltey) [reply app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsCardsItemFragment.kt:30]: Yes. Done.

**Pull request merged**